generators/app/templates/package.json

```diff
@@ -0,0 +1,116 @@
+{
+  "name": "ship-components-",
+  "version": "0.1.0",
+  "description": "",
+  "main": "src/index.js",
+  "scripts": {
+    "start": "grunt server",
+    "lint": "./node_modules/eslint/bin/eslint.js src/",
+    "test": "npm run lint && jest --coverage",
+    "test:update": "npm run test -- -u",
+    "test:watchAll": "npm run test -- --watchAll",
+    "watch:test": "jest --coverage --watch",
+    "coverage": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js -v",
+    "build": "npm test && grunt clean && grunt build",
+    "travis": "npm install trevor -g && trevor"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ship-components/generator-ship-components-boilerplate.git"
+  },
+  "engines": {
+    "node": ">=4.0 <7.0"
+  },
+  "keywords": [
+    "react",
+    "es2015",
+    "javascript",
+    "javascript-library",
+    "grunt",
+    "webpack",
+    "jest"
+  ],
+  "author": "Sepand Assadi <sepand.assadi@sony.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/ship-components/ship-components-tag-input/issues"
+  },
+  "homepage": "https://github.com/ship-components/ship-components-tag-input#readme",
+  "dependencies": {
+    "babel-runtime": "^6.23.0",
+    "bluebird": "^3.5.0",
+    "classnames": "^2.2.5",
+    "immutable": "^3.8.1",
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1",
+    "ship-components-icon": "^0.1.2",
+    "ship-components-outsideclick": "^0.2.0",
+    "ship-components-utility": "^1.2.1",
+    "superagent": "^3.5.2"
+  },
+  "devDependencies": {
+    "autoprefixer": "^7.1.1",
+    "babel-core": "^6.25.0",
+    "babel-eslint": "^7.2.3",
+    "babel-loader": "^7.1.1",
+    "babel-plugin-transform-runtime": "^6.23.0",
+    "babel-preset-env": "^1.6.0",
+    "babel-preset-react": "^6.24.1",
+    "coveralls": "^2.13.1",
+    "css-loader": "^0.28.4",
+    "enzyme": "^2.9.1",
+    "enzyme-to-json": "^1.5.1",
+    "eslint": "^4.1.1",
+    "eslint-config-ship-components": "^1.4.2",
+    "eslint-loader": "^1.8.0",
+    "eslint-plugin-react": "^7.3.0",
+    "extract-text-webpack-plugin": "^2.1.2",
+    "file-loader": "^0.11.2",
+    "grunt": "^1.0.1",
+    "grunt-contrib-clean": "^1.1.0",
+    "grunt-coveralls": "^1.0.1",
+    "grunt-eslint": "^20.0.0",
+    "grunt-webpack": "^3.0.0",
+    "jest": "^20.0.4",
+    "jest-css-modules": "^1.1.0",
+    "package-banner": "^0.1.1",
+    "postcss-calc": "^6.0.0",
+    "postcss-color-function": "^4.0.0",
+    "postcss-color-hex-alpha": "^3.0.0",
+    "postcss-loader": "^2.0.6",
+    "postcss-nested": "^2.0.2",
+    "postcss-simple-vars": "^4.0.0",
+    "prop-types": "^15.5.10",
+    "react-addons-test-utils": "^15.6.0",
+    "react-test-renderer": "^15.6.1",
+    "sockjs-client": "^1.1.4",
```

Comment:
Do we really need all of these dependencies? like sockjs? What is it used for? I don't want extra things in here because they will probably never be removed.

Reply:
Removed those that were not necessary